### PR TITLE
Lots of changes, mainly moving to exec task and adding nuget install

### PR DESCRIPTION
**Pull request by jonathanmorley**

### CODE REVIEW

File: src/main/groovy/com/ullink/BaseNuGet.groovy [hunk 1]
```diff
@@ -1,70 +1,46 @@
...
+        args '-NonInteractive'
+        args '-Verbosity', (verbosity ? verbosity : getNugetVerbosity())
 
-        project.exec {
-            commandLine = commandLineArgs
-        }
+        super.exec()
+    }
 
-        // NuGet return code is always 0 .... great ....
```
Comment (gluck) [inline src/main/groovy/com/ullink/BaseNuGet.groovy]: I see you dropped the command verification, is it because Nuget now returns a proper error code ?

Comment (jonathanmorley) [inline src/main/groovy/com/ullink/BaseNuGet.groovy]: Yes, I tested nuget pack on windows and mono, and got correct exit codes

Comment (gluck) [inline src/main/groovy/com/ullink/BaseNuGet.groovy]: Great !

**Comment by gluck** [comment]
:shipit:

**Pull request merged**